src/engine/engine.h

```diff
@@ -13,7 +13,16 @@ static constexpr audio::ChannelCount kMaxEngineChannelInputCount =
 // not.
 constexpr int kMaxSupportedStems = 4;
 #ifdef __STEM__
-constexpr uint kNoStemSelected = 0;
+enum class StemChannel {
+    First = 0x1,
+    Second = 0x2,
+    Third = 0x4,
+    Fourth = 0x8,
+
+    None = 0,
```

Comment:
Does None mean the stereo track?

Reply:
It looks like. At least it is zero if we have no stem file, right. Did you consider to implement https://github.com/mixxxdj/mixxx/issues/13700 right away. 
Is there a use case for "None". Probably in future so we may want to shift Everything by 1 to free up 0x01 for stereo and model the Stem file layout?

Reply:
Loading the premastered stereo track is clearly out of scope of this PR!
But it's possible to assign any unused value for it. The current implementation of 2^stemNo ensures simple understandable and unique stem masking accross the entire code base. 
Therefore I think this is the best possible code design here!

Reply:
> Does None mean the stereo track?
`None` means no stem track selected, so the deck will be loaded as stem deck, except for samplers, as they cannot load stems. `All` means load the stereo track

> Did you consider to implement #13700 right away.

Please see [this comment](https://github.com/mixxxdj/mixxx/pull/13268#issuecomment-2307726185) and [this one](https://github.com/mixxxdj/mixxx/pull/13268#issuecomment-2280504347). TL; DR: loading the pre-mastered stereo is not possible yet.

> Probably in future so we may want to shift Everything by 1 to free up 0x01 for stereo and model the Stem file layout?

I don't think we need to - AFAIK, there is 16 usecases, all covered by the flag:
- 0: load as stem deck
- 0..14: load as stereo with only the selected stems
- 15: load as stereo (currently mixed manually to address DSP limitation, using the pre-mastered track when we have the DSP)

Reply:
If "None" load as "stem deck" for a stem file and the stereo track in case of a normal file, we need to rename it to "Default" or such. 

Loading the stereo track IS a valid use case. 
Will it be used when all stems are selected or will we than just select all stems? 

This question is uncritical if we remove the exposal to the co interface from this PR.

Reply:
> If "None" load as "stem deck" for a stem file and the stereo track in case of a normal file, we need to rename it to "Default" or such.

Sounds fair, I will change that

> Loading the stereo track IS a valid use case.

it is

> Will it be used when all stems are selected or will we than just select all stems?

Currently, option 2 because no DSP support. In the future Option 1.